cpp/daal/src/algorithms/kernel_function/oneapi/kernel_function_rbf_dense_default_oneapi_impl.i

```diff
@@ -96,6 +96,11 @@ services::Status KernelImplRBFOneAPI<defaultDense, algorithmFPType>::computeRBF(
 
     KernelArguments args(6, status);
     DAAL_CHECK_STATUS_VAR(status);
+
+    DAAL_ASSERT_UNIVERSAL_BUFFER(sqrMatRight, algorithmFPType, n);
+    DAAL_ASSERT_UNIVERSAL_BUFFER(sqrMatRight, algorithmFPType, m);
+    DAAL_ASSERT(rbf.size() == n * m);
```

Comment:
overflow check?

Reply:
1. It should checked [here](https://github.com/oneapi-src/oneDAL/blob/master/cpp/daal/src/algorithms/kernel_function/kernel_function_fpt.cpp#L55)
2. This is in assert check - I would not want to add extra checks for the release build.

Reply:
How you can guarantee that `n` and `m` are always equal to `nVectors2` and `nVectors1`?

Reply:
Because sizes of `matLeftBuf` is equal `nVectors1` and `sqrMatRight` is equal `nVectors2` . Is always